rfcs/0016-verification/0016-verification.md

```diff
@@ -0,0 +1,117 @@
+---
+Number: "0016"
+Category: Informational
+Status: Draft
+Author: Dingwei Zhang
+Organization: Nervos Foundation
+Created: 2019-03-08
+---
+
+# CKB-Verification
+
+## Abstract
+
+This RFC describes ckb block data verification rules.
+
+Verification rules are split into three part in CKB implementation. `Header Verification`,  `Block Verification` and `Transaction Verification`.
+
+## Specification
+
+1. Header Verification
+
+    1. proof-of-work verification
+
+        Block to be valid it must hash to a value less than the current target. current hash algorithm is Cuckoo Cycle.
+
+    2. number verification
+
+        CKB block header contain block number, it MUST equal parent’s block number incremented by one, the genesis block has a number of zero.
+
+    3. block version verification
+
+        Block version MUST obey consensus rules.
```

Comment:
This is too vague. This rfc is defining consensus rules.

You can say: Block version MUST be zero.